utils/okta.py

```diff
@@ -606,8 +630,183 @@ def create_linked_users(self, userid, parentid, name):
 
         return RestUtil.execute_put(url=url, body=None, headers=okta_headers)
 
-    def create_clientcredential_application(self, app_name):
-        self.logger.debug("OktaAdmin.get_applications_by_id(app_id)")
+    def create_auth_server(self, auth_server_name, app_server_description):
```

Comment:
So the Okta Python SDK couldn't help here? I thinking it would reduce the code count.

Reply:
This would require a complete rework and weeks of testing/fixing.

Reply:
Gotcha. Okay.